docs/docs/cmd/spo/listitem/listitem-roleinheritance-reset.md

```diff
@@ -0,0 +1,39 @@
+# spo listitem roleinheritance reset
+
+Restores the role inheritance of list item, file, or folder
+
+## Usage
+
+```sh
+m365 spo listitem roleinheritance reset [options]
+```
+
+## Options
+
+`-u, --webUrl <webUrl>`
+: URL of the site from which the item role inheritance should be reset
+
+`--id <id>`
```

Comment:
Following our naming convention, this option should be named `listItemId` so indicate that we're expecting the ID of the list item. Without it, it would implicitly refer to roleinheritance ID which isn't the case.